utils/scarb-ui/src/components/spinner.rs

```diff
@@ -33,13 +34,27 @@ impl Drop for SpinnerHandle {
     }
 }
 
+impl PrintOverrider for ProgressBar {
```

Comment:
it's a tricky game to abuse submodule relationships, to workaround orphan rule, to write a trait impl that is a public API

was something off with the private newtype that provided this impl?